crates/matrix-sdk/src/event_cache/mod.rs

```diff
@@ -213,7 +218,19 @@ impl EventCache {
                 client.subscribe_to_ignore_user_list_changes(),
             ));
 
-            Arc::new(EventCacheDropHandles { listen_updates_task, ignore_user_list_update_task })
+            let (tx, rx) = mpsc::channel(32);
```

Comment:
Does it mean we are bounded to 32 subscribers? It seems a bit low. Can we increase to 256?

Reply:
No, it's not exactly the number of subscribers, it's the number of entries in the channel. In the worst case, it's the number of subscribers who try to send a notification at the same time, but with he `try_send` mechanism used in the drop impl, that should be sufficient to handle them all in order.